launch.sh

```diff
@@ -30,7 +31,7 @@ FT_SMARTNIC_CLIENT=false
 manage_labels
 
 if [ "$FT_SMARTNIC_SERVER" == true ] || [ "$FT_SMARTNIC_CLIENT" == true ]; then
```

Comment:
My only comment is that I don't see a reason to change the name of the net-attach-def unless you are pointing to one created outside of the scripts. In that case, might need a way to not create the one built into the scripts, and don't need to change the name in the netAttachDef-sriov.yaml.j2. Merging as is because it doesn't break the default case, but I might rework this later.